cmd/minify/main.go

```diff
@@ -735,7 +747,13 @@ func minify(t Task) bool {
 	} else {
 		// rename original when overwriting
 		for i := range t.srcs {
-			if t.srcs[i] == t.dst {
+			sameFile, err := SameFile(t.srcs[i], t.dst)
+			if err != nil {
```

Comment:
Had to remove this part in case the output file doesn't exist. In case of an error we should continue execution